mvpa2/algorithms/benchmarks/hyperalignment.py

```diff
@@ -0,0 +1,129 @@
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+#
+#   See COPYING file distributed along with the PyMVPA package for the
+#   copyright and license terms.
+#
+### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+"""Benchmarks for hyperalignment algorithms
+"""
+
+__docformat__ = 'restructuredtext'
+
+import numpy as np
+from mvpa2.base import warning
+from mvpa2.support import copy
+from mvpa2.mappers.zscore import zscore
+from mvpa2.generators.partition import NFoldPartitioner, HalfPartitioner
+from mvpa2.generators.splitters import Splitter
+from mvpa2.mappers.boxcar import BoxcarMapper
+from mvpa2.datasets.base import FlattenMapper
+from mvpa2.measures.anova import vstack
+from mvpa2.mappers.fx import mean_group_sample
+if __debug__:
+    from mvpa2.base import debug
+
+def zero_out_offdiag(dist, window_size):
+    for r in range(len(dist)):
+        dist[r, max(0, r-window_size):r] = np.inf
+        dist[r, r+1:min(len(dist), r+window_size)] = np.inf
+    return dist
+
+def _zero_out_offdiag(dist, window_size):
```

Comment:
Why two functions zero_out_offdiag, one with and the other without leading underscore? Do they do the same thing?